AO landing styles

noahmanger opened the pull request:
Adds a couple styles to support the new AO landing page in https://github.com/18F/openFEC-web-app/pull/1731:
- Styles for the titles of pending AOs
- Styles for the little document link
- Adjusts the sizing of the search container

jenniferthibault commented:
👍 May want technical review from @xtine

xtine commented on scss/components/_options.scss:
```diff
@@ -41,6 +41,12 @@
 
+.option__content {
+  .label {
+    margin-bottom: 2rem;
```
`u(2rem)`?

xtine commented on scss/components/_contact-items.scss:
```diff
@@ -1,6 +1,18 @@
+      content: '';
+      display: block;
+      background-size: 90%;
+      margin-right: 2rem;
```
Just noticed a few more units not in the `u` function in these styles.

noahmanger replied:
Fixed!

xtine commented on scss/components/_posts.scss:
```diff
@@ -2,6 +2,7 @@
 .post-feed {
   border-top: 1px solid $gray;
+  border-bottom: 1px solid $gray;
   padding: u(1rem 0 1rem 2rem);
```
I made a note in https://github.com/18F/openFEC-web-app/pull/1731 about the `padding-left` indent. Just want to make sure if it's intentional or needs a fix.

noahmanger replied:
Yep, that's the style we're going with as it's used on the press landing page too.

noahmanger commented:
Fixed merge conflicts. We were no longer using the `.icon-heading--calendar-circle` class elsewhere, so I removed it and un-nested the one you made for the home page.
